javascript/src/seedsource/components/Layers.jsx

```diff
@@ -2,32 +2,79 @@ import React from 'react'
 
 
 class Layers extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            displayResults: true,
+            displayVariables: true,
+            displaySeedZones: false
+        }
+    }
+
     render() {
         let { onToggleLayer, layers } = this.props
+
         let layerList = (urlSegment) => layers.filter(layer => layer.urlTemplate.includes(urlSegment))
             .map(layer => {
                 return (
-                    <div key={layer.name}>
+                    <li className="layer-list" key={layer.name}>
                         <input
+                            className="is-checkradio"
                             type="checkbox"
                             value={layer.name}
-                            onChange={() => onToggleLayer(layer.name)}
                             checked={layer.displayed}
                             disabled={(layer.urlTemplate === "seedZone") ? true : false}
-                        />
-                        {layer.name}
-                    </div>
+                        /><label onClick={() => onToggleLayer(layer.name)}>{layer.name}</label>
+                    </li>
                 )
             })
-        let [resultsLayer, seedZoneLayers, variableLayers] = ["{serviceId}", "seedZone", "{region}_{modelTime}"]
+
+        let [resultsLayer, variableLayers, seedZoneLayers] = ["{serviceId}", "{region}_{modelTime}", "tiles"]
             .map(layer => layerList(layer))
 
         return (
             <div className={"layers-tab"}>
-                {layers.length ? null : <p>No layers available</p>}
-                {resultsLayer.length ? <div><h4 className="title">Results</h4>{resultsLayer}<br/></div> : null }
-                {seedZoneLayers.length ? <div><h4 className="title">Seed Zones</h4>{seedZoneLayers}<br/></div> : null }
-                {variableLayers.length ? <div><h4 className="title">Variables</h4>{variableLayers}</div> : null }
+                <aside className="menu">
```

Comment:
Why `<aside>`? There only one child element.

Reply:
Actually seems that <aside> isn't any different in rendering than <div>: https://stackoverflow.com/questions/24690160/html-aside-tag-vs-div-tag